config/database.yml

```diff
@@ -14,12 +8,11 @@ development:
   encoding: unicode
   database: spacex
   port: 5432
-  pool: 5
-
+  pool: 100
 
 test:
   adapter: postgresql
   port: 5432
   encoding: unicode
   database: spacex
```

Comment:
Your test database has the same name as the development database. That is a grave mistake. It will create errors when running the test and it might overwrite all development data when running the tests successfully. Usually you have `spacex_development` for the dev environment and `spacex_test` for the test environment.